setup.py

```diff
@@ -559,6 +588,22 @@ def run(self, *args, **kwargs):
             produce_compile_commands_json=self.compile_commands,
         )
 
+class build_libpg_query(setuptools.Command):
+
+    description = "build libpg_query"
+
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        if os.environ.get("EDGEDB_BUILD_PACKAGE"):
```

Comment:
Remove this block.  `EDGEDB_BUILD_PACKAGE` indicates a production package (Postgres and the CLI are built separately in prod workflow because they require special handling, whereas libpq_query likely doesn't).